core/templates/dev/head/domain/exploration/SubtitledHtmlObjectFactorySpec.js

```diff
@@ -27,12 +27,13 @@ describe('SubtitledHtml object factory', function() {
       shof = $injector.get('SubtitledHtmlObjectFactory');
       subtitledHtml = shof.createFromBackendDict({
         html: '<p>some html</p>',
-        audio_translations: [{
-          language_code: 'en',
-          filename: 'a.mp3',
-          file_size_bytes: 20,
-          needs_update: false
-        }]
+        audio_translations: {
+          en: {
```

Comment:
Oh wait I see--it doesn't inherently have it in the backend, but when creating the State objects in the front-end the key is inserted as a value as well. I suppose this is just for convenience?